cmake/Modules/FindBLAS.cmake

```diff
@@ -172,6 +199,19 @@ if((NOT BLAS_LIBRARIES) AND (WIN32)
   endif(BLAS_LIBRARIES)
 endif()
 
+if((NOT BLAS_LIBRARIES) AND (WIN32)
```

Comment:
why is the WIN32 special case needed here?

Reply:
yes agreed and will remove